src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -227,6 +227,8 @@
    :refresh
    (fn [{:keys [props state]}]
      ((:request-refresh props))
+     (swap! state assoc :attrs-list
+            (vec (dissoc (get-in props [:workspace "workspace" "attributes"]) "description")))
```

Comment:
The refresh is async, so there's a chance that the attributes have changed and this is inaccurate.